src/Network/Probe.php

```diff
@@ -933,7 +933,12 @@ public static function profile($profile_link)
 		}
 
 		$prof_data = [];
-		$prof_data["addr"]         = $data["addr"];
+
+		// The "addr" is not always part of the fetched data
+		if (!empty($data["addr"])) {
+			$prof_data["addr"] = $data["addr"];
```

Comment:
This means the `addr` key of the `$prof_data` array may or may not be set from now on. This is inconsistent with the previous behavior which would have left `$prof_data["addr"]` set to `null` by default if `$data["addr"]` wasn't set. Please use `defaults()` instead.

Reply:
I'm unsure if we possibly would overwrite an existing "addr" with an empty value by using "defaults".

Reply:
No, the `$prof_data` array is initialized on line 935, there's no prior value.